README.md

```diff
@@ -38,6 +39,10 @@ Both data paths require the installation of the Thingworx Project Component, but
 
 The Thingworx project component must be installed and configured for both data paths.
 
+### Required Thingworx Version
```

Comment:
Is required version misleading? Would it be more appropriate to state "verified against". Should this state ThingWorx Foundation?

Reply:
I think it should stay as 'required version'. The reason for this additional commit was actually due to feedback that it was unclear what version the connector was made for, and it fails to install on versions previous than what is specified here (it's in Entities.xml at the top).